src/Popover/Popover.js

```diff
@@ -33,7 +36,7 @@ type Props = {
   /** Disables the Popover */
   disabled?: boolean,
   /** Include an arrow on the Popover content pointing to the trigger */
-  hasArrow?: boolean,
+  hasArrow?: boolean, // TODO: this prop seems to mess up the styles when set to false
```

Comment:
Can always handle this in a followup story if there is no apparent solution at present.  This issue already exists in Popover, no?